### ISSUE TITLE
Reply only takes current mailing list into account

### ISSUE THREAD

**Issue by sebbASF** [quenda/ponymail#54]
The various mail views only seem to show the current mailing list.

This causes a problem when using the Reply function; Reply All is usually what is needed.

There should at least be some indication that the e-mail was sent to more than 1 list.
[I know that the mail source is available, but that is hardly user-friendly]

**Comment by ilgrosso** [comment]
+1
And what if the e-mail features other (e.g. non mailing list) addresses?

**Comment by Humbedooh** [comment]
how would you detect a "non-mailing list" address though? Surely you'd want the system to just be interactions with lists, and not a generic email client for people. That's what they have their own email clients for.

**Comment by ilgrosso** [comment]
Ok, but what if one wants to reply to a specific e-mail which was sent to the mailing list AND someone (not subscribed to the mailing list, naturally)? Currently, the latter recipient won't get any reply - if replying via PonyMail.

**Comment by Humbedooh** [comment]
replying to both the list AND the original sender would be possible within the current design. It's just replying to all 'To: ' addresses that would not be possible.

**Comment by ilgrosso** [comment]
Sorry, I wasn't clear: I exactly mean this, e.g. replying to all `To` and `CC`.

**Comment by Humbedooh** [comment]
that would require rewriting the archiver to also digest those headers

**Comment by ilgrosso** [comment]
I agree that building a complete mail client is not on purpose, but if A sends an e-mail to mailing list M and additional recipient R, I would like to have Pony Mail sending out my reply to A's e-mail to both M and R.

**Comment by Humbedooh** [comment]
I'm not _against it_, but if this was added, it should be something that you enable on purpose (i.e. a setting that is off by default, but can be enabled). I feel the default should be 'reply to list' here. Anything beyond that would be an added 'nice to have'.

**Comment by ilgrosso** [comment]
LGTM :+1:

**Comment by Humbedooh** [comment]
welp, I must admit I was wrong. Pony Mail already saves the to/cc headers in the digest, so this is indeed possible with the current design.
